java-core/google-cloud-core/src/test/java/com/google/cloud/testing/VersionTest.java

```diff
@@ -39,57 +39,57 @@ public void testFromString() {
   }
 
   @Test
-  public void testFromStringWithAlphas() {
+  void testFromStringWithAlphas() {
     try {
       Version.fromString("2016.01.hello");
-      Assert.fail();
+      Assertions.fail();
     } catch (IllegalArgumentException ex) {
       assertNotNull(ex.getMessage());
     }
   }
 
   @Test
-  public void testFromStringMissingPatch() {
+  void testFromStringMissingPatch() {
     try {
       Version.fromString("2016.01");
-      Assert.fail();
+      Assertions.fail();
     } catch (IllegalArgumentException ex) {
       assertNotNull(ex.getMessage());
     }
   }
 
   @Test
-  public void testFromStringMissingMinor() {
+  void testFromStringMissingMinor() {
     try {
       Version.fromString("2016");
```

Comment:
Same with these. Maybe we can use `assertThrows()`?

Reply:
Done.